charts/avalanche/values.yaml

```diff
@@ -6,9 +6,17 @@ image:
   tag: main
   pullPolicy: IfNotPresent
 
-# number of connector pods to spawn
+# Number of Avalanche pods to spawn. Majorly useful to test Avalanche with
+# remote_write.
+# Use fakePrometheusTargetCount when using Avalanche with
+# Prometheus to generate load.
 replicaCount: 1
 
+# Number of fake PodMonitor to be created to simulate fake Prometheus
+# targets. This will be useful to generate more series without scaling
+# Avalanche pods when using it as a Prometheus target.
+fakePrometheusTargetCount: 0
```

Comment:
I see that this is not backward compatible, Since there is no ServiceMonitor, pulling this change would break the update. Let me fix it.